app/views/pages/list-school-job.html.slim

```diff
@@ -0,0 +1,229 @@
+          | You can shortlist or reject applications quickly and easily, and can print or download the applications.
+
+      .govuk-grid-column-one-half class="govuk-!-padding-left-0"
+        img class="govuk-!-margin-top-4" src=asset_pack_path("media/images/hiring_staff_landing_page/manage_applicants.png")
+
+    hr.govuk-section-break.govuk-section-break--l.govuk-section-break--visible class="govuk-!-margin-top-2 govuk-!-margin-bottom-2"
+
+    .govuk-grid-row.section class="govuk-!-padding-left-3"
+      h3.govuk-heading-m class="govuk-!-margin-top-4"
+        | Who's using Teaching Vacancies?
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        h4.govuk-heading-l class="govuk-!-font-size-80 govuk-!-margin-top-4"
+          | 81%
+        p.govuk-body
+          | of schools in England
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        h4.govuk-heading-l class="govuk-!-font-size-80 govuk-!-margin-top-4"
+          | 2.2
+        strong.govuk-body class="govuk-!-font-weight-bold"
+          | million
+        span.govuk-body
+          |  vacancy views in 2021
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        h4.govuk-heading-l class="govuk-!-font-size-80 govuk-!-margin-top-4"
+          | 1.5
+        strong.govuk-body class="govuk-!-font-weight-bold"
+          | million
+        span.govuk-body
+          |  job alert emails sent in 2021
+
+    hr.govuk-section-break.govuk-section-break--l.govuk-section-break--visible class="govuk-!-margin-top-2 govuk-!-margin-bottom-2"
+
+    .govuk-grid-row.section class="govuk-!-padding-left-3"
+      h3.govuk-heading-m class="govuk-!-margin-top-4 govuk-!-margin-bottom-6"
+        | What schools say about Teaching Vacancies
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        p.govuk-body
+          | “Using Teaching Vacancies was free and simple to set up… and reached a wider group of applicants than our
+          |   usual advertising.”
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        p.govuk-body
+          | “What we have saved is enough to recruit another teacher!”
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        p.govuk-body
+          | “We like that we are totally in control and can post directly.”
+
+    hr.govuk-section-break.govuk-section-break--l.govuk-section-break--visible class="govuk-!-margin-top-2 govuk-!-margin-bottom-2"
+
+    .govuk-grid-row.section class="govuk-!-padding-left-3"
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        h3.govuk-heading-m class="govuk-!-margin-top-4"
+          | Sign in
+
+        = govuk_link_to("Sign in and list your jobs for free", publishers_sign_in_path, class: "govuk-body-m govuk-!-margin-bottom-4")
+
+        = govuk_link_to("Request a Teaching Vacancies account", publishers_account_requests_new_path, class: "govuk-body-m govuk-!-margin-bottom-4")
+
+    .spacer
```

Comment:
why not just a margin or padding bottom?

Reply:
The largest spacing override is `govuk-!-margin/padding-bottom-9` as far as I'm aware. It's not big enough.